app/models/webhook.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class Webhook < ApplicationRecord
+  enum status: {
+    pending:   'pending',
+    processed: 'processed',
+    error:     'error'
+  }
+
+  def process
+    processed = case gateway
+                when 'paypal'
+                  License.find_by(subscription_id: reference_id)&.check_subscription
+                end
+
+    update! status: (processed ? :processed : :error)
```

Comment:
Ya que estamos, acá podría ser:
```suggestion
    processed ? processed! : error!
```